### ISSUE TITLE
Concern over code action arrow - UI

### ISSUE THREAD

**Issue by justineechen** [OpenLiberty/openliberty.io#1239]
There has been some discussion over the UI green arrow and how we word instructions in this arrow. 

@aguibert 

<img width="477" alt="Screen Shot 2019-06-25 at 11 58 38 AM" src="https://user-images.githubusercontent.com/18709242/60113915-bf9dc600-9740-11e9-8197-3134e2e7f702.png">

<img width="476" alt="Screen Shot 2019-06-25 at 11 58 27 AM" src="https://user-images.githubusercontent.com/18709242/60113914-bf9dc600-9740-11e9-8ef4-5865f2a2fefe.png">

**Comment by yeekangc** [comment]
@mdenby @cajaygle

**Comment by steven1046** [comment]
I agree with just saying 'Create the xyz file'

**Comment by mbroz2** [comment]
Once feedback has been provided by design, this will likely get moved to guides/common repo for the guide authors to make any textual changes (if necessary, per design, we'll make any css changes)

**Comment by mdenby** [comment]
I'm fine with:
<img width="855" alt="Screen Shot 2019-07-10 at 4 49 43 PM" src="https://user-images.githubusercontent.com/26823018/61003881-c3ca0600-a332-11e9-9ff8-ccb44f15b3b9.png">

Does Lauren need to look at this too? And who is going back through all the guides to find these instances?

**Comment by yeekangc** [comment]
Right, once we settle on a design and it's implemented/available, we may need to update all guides to adopt it.  @gkwan-ibm will need to plan and coordinate then.  Then, if the current metadata suffices, it may be simply a change on the UI site without needing any content updates.

Yes, we will need the UI team to look and chime in.

May be it should be "Location:" (as Andy suggested) or "File location:" so there is no ambiguity.

**Comment by mbroz2** [comment]
Sentance description of action
File Location: 

Create the dockerfile
File:

**Comment by yeekangc** [comment]
If the below is simply for applying the style, then this will be content change only once we settle on the wordings etc.

```
[role="code_command hotspot", subs="quotes"]
----
#Create the `EndpointTest` class.#
`src/test/java/it/io/openliberty/guides/rest/EndpointTest.java`
----
```

**Comment by mbroz2** [comment]
@yeekangc unless there's further concern, we can use the wording above.  
I expect there's no need for any code changes to support prepending "File:" before the path, but @steven1046 can confirm:
```
[role="code_command hotspot", subs="quotes"]
----
#Create the `EndpointTest` class.#
File: `src/test/java/it/io/openliberty/guides/rest/EndpointTest.java`
----
```
in which case we'd only need to update the guides adoc files; I think the only exception would be if we wanted to style "File:" in a particular manner

**Comment by steven1046** [comment]
Right, we can add `File: ` in the .adoc files. If we need to style it then we will need to edit the css for guides.

**Comment by yeekangc** [comment]
I lean toward "File location" to be clear as to what it is.

@steven1046, what determines which file is highlighted by the arrow?  Doesn't look like it's this "code_command hotspot"?

**Comment by steven1046** [comment]
If it has the role `hotspot`, then it will switch to the first file in that section unless `file=#` is added to the role as well to indicate a different file in that subsection.

**Comment by yeekangc** [comment]
Looks like this is content change then.  We can move the issue over to guides-common for prioritization.

I have thoughts on relying on the order of the files here.  Will discuss with the team and raise a separate issue if necessary.
